module/VuFind/src/VuFind/Db/Service/SearchService.php

```diff
@@ -61,7 +62,8 @@ class SearchService extends AbstractDbService implements
      */
     public function createEntity(): SearchEntityInterface
     {
-        return $this->getDbTable('search')->createRow();
+        $class = $this->getEntityClass(Search::class);
```

Comment:
I think we should use the interface here rather than the absolute class. (I realize this is inconsistent in some of the other services -- there's a TODO checkbox on #2233 to normalize everything to use the interfaces).

```suggestion
        $class = $this->getEntityClass(SearchEntityInterface::class);
```